src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -204,10 +204,10 @@ protected void LookupMembersInternal(LookupResult result, NamespaceOrTypeSymbol
             compatibleExtensions.Free();
             return;
 
-            void getCompatibleExtensions(Binder binder, TypeSymbol type, ArrayBuilder<NamedTypeSymbol> compatibleExtensions, ConsList<TypeSymbol>? basesBeingResolved)
+            static void getCompatibleExtensions(Binder binder, TypeSymbol type, ArrayBuilder<NamedTypeSymbol> compatibleExtensions, Binder originalBinder, ConsList<TypeSymbol>? basesBeingResolved)
             {
                 var extensions = ArrayBuilder<NamedTypeSymbol>.GetInstance();
-                binder.GetImplicitExtensionTypes(extensions, originalBinder: this);
+                binder.GetImplicitExtensionTypes(extensions, originalBinder);
```

Comment:
>originalBinder [](http://example.com/codeflow?start=61&length=14)

Is this change observable? If so, is it backed by a test?

Reply:
There is no change as previously callers would use the same value for receiver and for `originalBinder`:
`binder.LookupImplicitExtensionMembersInSingleBinder(..., scope.Binder, ..., originalBinder: binder, ...);`
`LookupImplicitExtensionMembersInSingleBinder(..., scope.Binder, ..., originalBinder: this, ...);`

Now, we use `this` for the current binder (instead of a specific parameter) and the first caller still uses `originalBinder: binder` and the second caller still uses `originalBinder: this`.

That said, the logic related to `originalBinder` is not yet covered. We have a PROTOTYPE comment in `GetImplicitExtensionTypes`:
```
            // PROTOTYPE test this flag (see TestUnusedExtensionMarksImportsAsUsed)
            bool callerIsSemanticModel = originalBinder.IsSemanticModelBinder;
```